test/metabase/cmd/load_from_h2_test.clj

```diff
@@ -48,3 +58,93 @@
        (testing "H2 connection details SHOULD have been copied"
          (is (=? {:db string?}
                 (t2/select-one-fn :details :model/Database :engine :h2))))))))
+
+(defn- get-data-source [db-type db-def]
+  (let [connection-details (tx/dbdef->connection-details db-type :db db-def)
+        db-spec (sql-jdbc.conn/connection-details->spec db-type connection-details)]
+    (mdb.test-util/->ClojureJDBCSpecDataSource db-spec)))
+
+(defn- create-current-database
+  [db-type db-def data-source]
+  (tx/create-db! db-type db-def)
+  (mdb.setup/setup-db! db-type data-source true))
+
+(defn- dump-filename
+  [h2-filename version]
+  (str h2-filename "-" version))
+
+(defn- parse-major-version
```

Comment:
```suggestion
(defn- major-version
```

this doesn't take any argument, and prefix with `parse` makes it feel like it should process something.

I suggest removing the prefix

Reply:
also wdyt about naming this `liquibase-latest-major-version` to make it explicit where it comes from?